fadecandy_driver/CMakeLists.txt

```diff
@@ -1,14 +1,61 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 3.0.2)
 project(fadecandy_driver)
 
-find_package(catkin REQUIRED)
+add_compile_options(-std=c++11)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall")
 
-catkin_python_setup()
+##################
+##  Build setup ##
+##################
 
-catkin_package()
+find_package(catkin REQUIRED COMPONENTS
+  diagnostic_updater
+  fadecandy_msgs
+  roscpp
+)
 
-catkin_install_python(
-    PROGRAMS
-    scripts/fadecandy_node
-    DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
-    )
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/")
```

Comment:
While building fadecandy_driver catkin package, CMake is failing to find where Findlibusb.cmake file is located. If the directory structure is like the following, then we may have to incorporate `PROJECT_NAME` into `CMAKE_MODULE_PATH` like this: `set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/${PROJECT_NAME}/cmake/")`. Please let me know what you think?

```sh
workspace/
   | - build/
   | - devel/
   | - src/
        | - fadecandy_driver/
            | - cmake/
            | - include/
            | - src/
            | - CMakeLists.txt
            | - package.xml
```

Reply:
Updated the CMAKE file + submodule: Instead of a custom findlibusb, we use `pkgconf` now. Let me know whether this works for you.

Reply:
Great. Will check it out. Thanks

Reply:
@reinzor `pkg_check_modules` change worked. Now testing and reviewing the driver. Thanks for your contribution!